src/components/game/GameEndingBoard/index.js

```diff
@@ -0,0 +1,81 @@
+import React, { useState, useCallback } from "react";
+import { useTranslation } from "react-i18next";
+import { Header, Segment, Card, Button, Icon, Dimmer } from "semantic-ui-react";
+
+import Avatar from "components/user/Avatar";
+import { lobbyClient } from "services/LobbyClient";
+import DataStore from "services/DataStore";
+
+import { useUser } from "contexts/UserContext";
+import { useHistory } from "react-router-dom";
+import { routes } from "config/routes";
+
+import { useBoardGame } from "contexts/BoardGameContext";
+import Loading from "components/game/Loading";
+
+const GameEndingBoard = ({ winners, players }) => {
+  const { t } = useTranslation(["translation", "lobby"]);
+  const { playerID, matchID, gameName, credentials } = useBoardGame();
+  const [loading, setLoading] = useState(false);
+
+  const user = useUser();
+  const history = useHistory();
+
+  const handleNewMatch = useCallback(() => {
+    setLoading(true);
+    lobbyClient
+      .playAgain(gameName, matchID, { playerID: playerID, credentials: credentials })
```

Comment:
Nice 👍

Reply:
It's been there since v0.39.10. We must have missed it somehow 😄